L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -89,8 +89,8 @@
 # Barrel and EndCap EGamma
 # ########################################################################
 
-from L1Trigger.L1CaloTrigger.l1tEGammaCrystalsEmulatorProducer_cfi import *
```

Comment:
@aloeliger replacing the old emulator with the new one has knock-on effects on downstream consumers: the correlator consumes the old collection for PF and Egamma